src/lib/build-style.js

```diff
@@ -215,20 +205,19 @@ const buildLayer = (context, name, path) => {
  * @param {string} layerDir - the input directory that contains layers
  * @returns {Object}
  */
-export const buildStyle = (name, styleDir, layerDir, options = {}) => {
-  if (!name) {
-    throw new Error('Must provide name.');
-  }
-  if (!styleDir) {
-    throw new Error('Must provide styleDir.');
+export const buildStyle = (stylePath, layerDir, options = {}) => {
```

Comment:
Everything above here in this file is an unintentional formatting change. Happy to try and remove, but since the changes are pretty small, I figured I'd just leave this comment.